less/pages/team.less

```diff
@@ -1,56 +1,26 @@
 .team-page {
+  background: #fff;
 
-}
-
-.member-profiles {
-  display: flex;
-  flex-direction: column;
-  margin: -10px 0 0 -10px;
-  flex-wrap: wrap;
-  @media screen and (min-width: 768px) {
-    flex-direction: row;
+  .hero-unit-container + .content {
+    overflow: visible;
+    h1 {
+      margin-top: 2rem;
+    }
+  }
+  .tab-switcher.pull-up {
+    margin-top: -4.93rem;
```

Comment:
The tabs break (text disappears) on mobile. In Science they use icons, we might just want smaller text to avoid wrap.